source/manuals/programming-languages/html.html.md.erb

```diff
@@ -1,85 +1,168 @@
-still good to use:
+`<header role="banner">` should be used to contain your home link, branding, search,
+and any global navigation you may have.
 
-* `role="search"`
-* `role="application"`
+### Main content
 
-[Demo of how landmark roles help screen reader users](http://tink.co.uk/2011/07/how-do-aria-landmark-roles-help-screen-reader-users/)
+`<main role="main">` should be used to identify the main content of your page.
 
-## Main content
+A "Skip to main content" link should be added to help screen reader and keyboard
+users skip past your general site navigation and get straight to the content.
+You can use the [GOV.UK skip link component](https://design-system.service.gov.uk/components/skip-link/)
+for this.
 
-The `<main>` tag should be used to identify the main content of your page. Support for it is still marginal so ensure to add the `role="main"` attribute.
+### Sectioned content
 
-## Sectioned content
+Use the `<section>` tag to represent standalone, thematic groupings of content
+where a more specific semantic element would not be appropriate.
 
-When using the `<section>` tag, add an `aria-labelledby` attribute linking to the id of the heading that identifies that section.
+Examples of sections would be chapters, or various tabbed pages in a tabbed dialog box,
+or a home page split into sections such as introduction, news items and contact information.
+
+You should add an `aria-labelledby` attribute linking to the id of the heading
+that identifies that section.
 
 Example:
 
-    <section aria-labelledby="advice">
-      <h1 id="advice">Help and advice</h1>
+```html
+<section aria-labelledby="introduction">
+    <h1 id="introduction">Introduction</h1>
+    <p>People have been catching fish for food since before recorded history...</p>
+</section>
 
-## Navigational elements
+<section aria-labelledby="equipment">
+    <h1 id="equipment">Equipment</h1>
+    <p>The first thing you'll need is a fishing rod or pole that you find comfortable
+    and is strong enough for the kind of fish you're expecting to try to land...</p>
+</section>
+```
 
-When using the `<nav>` tag, add the ARIA landmark `role="navigation"`.
+### Navigational elements
 
-The detail of using the `<nav>` tag is covered in this [blog post](https://insidegovuk.blog.gov.uk/2013/07/03/rethinking-navigation/).
+Use `<nav role="navigation">` to wrap groups of links that aren't already in
+another context (e.g. `<footer>`). Common examples of navigation sections are
+menus, tables of contents, and indexes.
 
-### Emphasis
+As with `<section>`, it is advisable to use `aria-labelledby` to label your `<nav>`.
+This is helpful in disambiguating where there are different `<nav>` blocks in the page.
 
-If you want to give the current location emphasis within a navigational list, use a `<strong>` tag.
+The detail of using the `<nav>` tag is covered in this [GOV.UK blog post](https://insidegovuk.blog.gov.uk/2013/07/03/rethinking-navigation/).
```

Comment:
Is there a way of rewording this sentence so that the link text is more descriptive? e.g. `The GOV.UK blog has some [more information on the use of the nav tag]` ?